Reword getting started wrt openshift-specifics (#1843)
strimzi/strimzi-kafka-operator: documentation/book/getting-started.adoc

[id='getting-started-{context}']
= Getting started with {ProductName}

> {ProductName} works on all types of Kubernetes cluster irrespective of distribution, from public and private clouds to local deployments intended for development.

Reviewer: Given you cannot test everything, should this say more something like _{ProductName} was designed to work on all types of Kubernetes clusters_?

Author: ```suggestion
{ProductName} is designed to work on all types of Kubernetes cluster irrespective of distribution, from public and private clouds to local deployments intended for development.
```

> {ProductName} is designed to work on all types of Kubernetes cluster irrespective of distribution, from public and private clouds to local deployments intended for development.

Reviewer 2: ```suggestion
{ProductName} is designed to work on all types of Kubernetes cluster regardless of distribution, from public and private clouds to local deployments intended for development.
```

> {ProductName} supports a few features which are specific to OpenShift, where such integration benefits OpenShift users and cannot be implemented equivalently using standard Kubernetes.

Reviewer: Should we now call it OKD in upstream instead of OpenShift? Or do we think OpenShift is ok here?

Author: I wonder how many people know what OKD is?

Reviewer 3: Yeah, the same feeling we don't say OKD either:-)

Author: ```suggestion
{ProductName} supports a few features which are specific to OpenShift/OKD, where such integration benefits OpenShift users and cannot be implemented equivalently using standard Kubernetes.
```

?

Reviewer 2: I suggest "specific to OpenShift Container Platform and OKD".

Author: ```suggestion
{ProductName} supports a few features which are specific to OpenShift and OKD, where such integration benefits OpenShift users and cannot be implemented equivalently using standard Kubernetes.
```

Reviewer: I don't think it makes sense to use both OpenShift and OKD. The point is the OpenShift is a name of commercial offering. OKD isn't.

Author: Mentioning OKD here, when other parts of the doc talk about OpenShift is inconsistent. We're we're going to change to OKD here, then why not everywhere?

Reviewer: That was of course a generic comment applying to all usages. And I agree that nobody knows what OKD is. I think saying _OpenShift and OKD_ is pointless.

I do not mind if OpenShift is used here. I just thought that if people complain that the docs are too OpenShift specific it is for sure also because it is a name of a commercial product. I thought one could mitigate that by using OKD. But if you think its not needed, lets keep using OPenShift

Author: I'm not opposed to changing it to OKD, but until someone objects specifically to the remaining OpenShift occurrences on the basis of it being commercial (so that changing it to OKD would remedy the situation) then I think it's just more helpful for the reader to leave it as OpenShift (since they're more likely to already know what that is).

> {ProductName} supports a few features which are specific to OpenShift and OKD, where such integration benefits OpenShift users and cannot be implemented equivalently using standard Kubernetes.

Author: ```suggestion
{ProductName} supports a few features which are specific to OpenShift, where such integration benefits OpenShift users and cannot be implemented equivalently using standard Kubernetes.
```

ifdef::InstallationAppendix[]
When no existing Kubernetes cluster is available, `minikube` can be used to create a local cluster.
More details can be found in xref:installing_kubernetes_cluster[Installing Kubernetes clusters].

> If you need to use OpenShift-specific features and have no OpenShift cluster `minishift` can be used to create a local cluster.

Reviewer: Nobody needs to use the OpenShift specific features - especially if they do not have an OPenShift cluster. Would this be better as _If you want ..._?

Author: ```suggestion
If you want to use OpenShift-specific features and have no OpenShift cluster `minishift` can be used to create a local cluster.
```
